dbt-athena/hatch.toml

```diff
@@ -0,0 +1,57 @@
+[version]
+path = "src/dbt/adapters/athena/__version__.py"
+
+[build.targets.sdist]
+packages = ["src/dbt/adapters", "src/dbt/include"]
+sources = ["src"]
+
+[build.targets.wheel]
+packages = ["src/dbt/adapters", "src/dbt/include"]
+sources = ["src"]
+
+[envs.default]
+dependencies = [
+    "dbt-adapters @ git+https://github.com/dbt-labs/dbt-adapters.git",
+    "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git",
+    "dbt-tests-adapter @ git+https://github.com/dbt-labs/dbt-adapters.git#subdirectory=dbt-tests-adapter",
+    "dbt-core @ git+https://github.com/dbt-labs/dbt-core.git#subdirectory=core",
```

Comment:
These are always pulling from main. Is that what we actually want?

Reply:
Yes, that's how we currently do it. Once all adapters are in this repo, I'll update to install locally so that it's always the current branch. I want to do that in a separate PR though.